ProffieOS.ino

```diff
@@ -18,14 +18,12 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-/*
----------------------------------------------------------------------
--  You can have multiple configuration files, and specify which one -
--  to use here by removing the comment slashes at the beginning.    -
--  **NOTE** Only ONE line should be left uncommented at a time!     -
--  Add the slashes to any that you are not uploading.               -
----------------------------------------------------------------------
-*/
+/*---------------------- ProffieOS:6.3 ----------------------------*\
```

Comment:
The version string on line 209 is updated automatically by the export script.
I don't want to have a version that I have to update manually.
IF we put the version here, it would look like $Id: ..... $ and when you check out the code from github you would only get a long hex string, not anything helpful. Also, the version string changes length, which would ruin all this fancy formatting....

Reply:
That's what i was referring to, yes. Ok cool, reverted.